tracing/hooks_test.go

```diff
@@ -19,7 +19,7 @@ type CallContainer struct {
 func (c *CallContainer) AddCall(call string, fail bool) error {
 	c.Calls = append(c.Calls, call)
 	if fail {
-		return fmt.Errorf(call)
+		return errors.New(call)
```

Comment:
does go vet call this out now?

Reply:
staticcheck calls this out now: https://github.com/reddit/baseplate.go/actions/runs/10407002603/job/28821319767#step:6:7

Reply:
the main point I think is the dynamic string part, so a string literal would still work, and if you use something like `fmt.Errorf("%s", call)` that likely also works.

Reply:
which I think makes sense, as there's a possibility `call` contains some format verbs and then `fmt.Errorf(call)` would cause an error.